### PR TITLE
Nimbus / Pan-M dataset plot notebooks

### PR DESCRIPTION
**Pull request by JLrumberger**
**What is the purpose of this PR?**

Adding the code for producing the plots for our upcoming paper.

**Remaining issues**

None

### CODE REVIEW

File: 2024-Rumberger_etal_Nimbus/README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Code for Rumberger et al., Improving cell phenotyping by factoring in spatial marker expression patterns with Nimbus
```
Comment (ngreenwald) [inline 2024-Rumberger_etal_Nimbus/README.md:1]:
```suggestion
# Code for Rumberger, Greenwald et al., Improving cell phenotyping by factoring in spatial marker expression patterns with Nimbus
```

Comment (ngreenwald) [reply 2024-Rumberger_etal_Nimbus/README.md:1]: Same for folder name please :)

**Review by ngreenwald — CHANGES_REQUESTED** [review]
Looks good, just the right level of description in the readme.

Figure 2 plotting notebook has the figure panels in kind of a confusing order, is this because the code for earlier panels depends on the later panels?

**Comment by JLrumberger** [comment]
I reordered the figure 2 plotting notebook, now it's more organized (b-f, then the overlay images a, g-i) and corrected the foldername / readme. I'll close this branch and open a new one, because I can't work on this repository locally. A few files in `2022-McCaffrey_etal_HumanTB` cause errors when cloning the repo. I tried downloading it as .zip, and then adding this repo as origin to the local repo, but I still got errors.

**Pull request closed without merging**